packages/react/src/alert-dialog/AlertDialog.css.ts

```diff
@@ -0,0 +1,78 @@
+import { theme } from "../styles";
+import { type RecipeVariants, recipe, style } from "../vanilla-extract";
+
+export const content = recipe({
+  base: [
+    {
+      bg: "white",
+      rounded: "lg",
+      shadow: "md",
+    },
+    style({
+      position: "absolute",
+    }),
+  ],
+  variants: {
+    size: {
+      sm: style({ width: "375px" }),
+      md: style({ width: "600px" }),
+      lg: style({ width: "800px" }),
+    },
+  },
+});
+
+export const overlay = recipe({
+  base: [
+    {
+      alignItems: "center",
+      bg: "dark.200",
+      size: "full",
+    },
+    style({
+      position: "absolute",
+    }),
+  ],
+});
+
+export const footer = recipe({
+  base: [
+    {
+      flexDirection: "row",
+      gap: "md",
+      justifyContent: "end",
+      px: "24",
+      py: "20",
+    },
+    style({
+      borderTop: `1px solid ${theme.colors["border.secondary"]}`,
+    }),
+  ],
+});
+
+export const title = recipe({
+  base: [
+    {
+      fontSize: "2xl",
+      fontWeight: "600",
+      pb: "16",
+      pt: "24",
+      px: "24",
+    },
+  ],
+});
+
+export const description = recipe({
+  base: [
+    {
+      fontSize: "md",
+      overflow: "auto",
+      px: "24",
+      py: "16",
+    },
+    style({
+      maxHeight: "50dvh",
```

Comment:
let's restrict the height of the `content` rather than the description - so we can accurately handle different screen sizes

Reply:
If we restrict the content, it has title and action buttons. they will also get scrolled vai, isn't it?

Reply:
Got it vai.

Reply:
let's address in a separate PR